nvidia/cheminformatics/data/helper/chembldata.py

```diff
@@ -12,17 +11,17 @@
 from contextlib import closing
 from nvidia.cheminformatics.utils.singleton import Singleton
 from nvidia.cheminformatics.smiles import RemoveSalt, PreprocessSmiles
-from nvidia.cheminformatics.fingerprint import MorganFingerprint, Embeddings
+from nvidia.cheminformatics.fingerprint import Embeddings
```

Comment:
I think the MorganFingerprint class needs to be imported but the Embedding class does not, correct?

Reply:
Embeddings is set as default. That requires importing it. The caller will import MorgenFingerprint, to change the default. Therefore this code does not require importing MorgenFingerprint.